docs/wire_compiler.md

```diff
@@ -63,7 +63,7 @@ code. It uses [KotlinPoet][kotlinpoet] internally to generate compact and legibl
 
 ![Basic](images/gradle_basic@2x.png)
 
-Wire will generate files in `build/generated/src/main/java`. It'll also register this directory as a
+Wire will generate files in `build/generated/source/wire`. It'll also register this directory as a
```

Comment:
kapt, apt use `build/generated/source`.
sqldelight use `build/sqldelight`.
the former seemed more conventional.

Reply:
I like `build/wire` more; isn't just beautifully simple?

Reply:
It's shorter, for sure. 
On the other hand, buildd/generated/source/wire vs build/generated/source/apt vs build/generated/source/kapt has a nice symmetric structure IMO.  Do you feel strongly either way?

Reply:
My personal preference is for `build/generated/source/wire`, but both are fine IMO.